includes/discovery/sensors/state/timos.inc.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$power_supply_table = end(snmpwalk_cache_numerical_oid($device, 'tmnxChassisPowerSupplyTable', $power_supply_table = array(), 'TIMETRA-CHASSIS-MIB', 'aos', '-OQUsn'));
```

Comment:
This can probably be refactored; it was being used to verify what was being grabbed, but then I ended up reusing the table for the population.  We can probably replace it with a direct `snmp_get` in the loop below.